eo-parser/src/main/resources/org/eolang/parser/errors/broken-aliases.xsl

```diff
@@ -28,7 +28,7 @@ SOFTWARE.
     <xsl:copy>
       <xsl:apply-templates select="node()|@*"/>
       <xsl:for-each select="/program/metas/meta[head='alias']">
-        <xsl:if test="not(matches(tail, '^([a-z]+ )?[\w\d]+(\.[\w][\w\d_-]*)*$'))">
+        <xsl:if test="not(matches(tail, '^([\w]+ )?[\w\d]+(\.[\w][\w\d_-]*)*$'))">
```

Comment:
@Graur this is not entirely correct. `\w` means `[a-zA-Z]`, but identifiers in EO can't start with `[A-Z]`. the first letter should always be small: `[a-z]`

Reply:
@Graur see https://github.com/objectionary/eo/blob/master/paper/sections/syntax.tex#L20-L27